src/components/chat/index.js

```diff
@@ -375,8 +376,8 @@ module.exports = {
             let future = peergos.shared.util.Futures.incomplete();
             let reader = new browserio.JSFileReader(mediaFile);
             let java_reader = new peergos.shared.user.fs.BrowserFileReader(reader);
-            if (mediaFile.size > 2147483647) {
-                that.showMessage("Media file greater than 2GiB not currently supported!");
+            if (mediaFile.size > 200 * 1024 * 1024) {
+                that.showMessage("Media file greater than 200 MiB not currently supported!");
```

Comment:
"Media files"... are not supported. Instead, you can upload a larger file to your drive and share it with a secret link here."